prospector-more.yml

```diff
@@ -1,48 +0,0 @@
-inherits: prospector
-
-strictness: high
-
-ignore-paths:
-  - constants.py
-  - urls.py
-  - wsgi.py
-  - acl
-  - api
-  - betterversion
-  - bookmarks
-  - builds
-  - cdn
-  - comments
-  - core
-  - djangome
-  - doc_builder
-  - donate
-  - gold
-  - integrations
-  - locale
-  - notifications
-  - oauth
-  - payments
-  - privacy
-  - profiles
-  - projects
-  - redirects
-  - restapi
-  - rtd_tests
-  - search
-  - settings
-  - tastyapi
-  - templates
-  - vcs_support
```

Comment:
Hrm, this might be a file to leave out of the common repo for now. These includes are all specific to RTD